src/Phan/Language/Element/PassByReferenceVariable.php

```diff
@@ -144,4 +144,12 @@ public function isPHPInternal(): bool
     {
         return $this->element instanceof AddressableElement && $this->element->isPHPInternal();
     }
+
+    /**
+     * @suppress PhanUnreferencedPublicMethod this may be called by plugins
```

Comment:
`src/Phan/Language/Element/PassByReferenceVariable.php:151 PhanPluginDescriptionlessCommentOnPublicMethod Public method \Phan\Language\Element\PassByReferenceVariable::getParameter has no readable description: "* @suppress PhanUnreferencedPublicMethod this may be called by plugins"`
```suggestion
     * Get the parameter that this PassByReferenceVariable was passed into.
     * @suppress PhanUnreferencedPublicMethod this may be called by plugins
```